roles/sos_report/tasks/sos-reports.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Generate SOS report for node {{ node_name }}
+  vars:
+    sos_report_registry: "{{ sos_report_image | regex_replace('([^/]+)/.*', '\\1') }}"
+    sos_report_image_name: "{{ sos_report_image | regex_replace('[^/]+/(.*)', '\\1') }}"
+  shell: >
+    {{ sos_report_oc_path }} debug --image={{ sos_report_image }} node/{{ node_name }} --
+    bash -c 'echo -e REGISTRY='{{ sos_report_registry }}'\\nIMAGE='{{ sos_report_image_name }}'> /host/root/.toolboxrc';
+    {{ sos_report_oc_path }} debug --image={{ sos_report_image }} node/{{ node_name }} --
```

Comment:
why do you need to run the shell cmd into 2 different 'oc debug' instead of one? Is it not possible to add the .toolboxrc and run the 'toolbox' cmd at the same time?

Reply:
certainly can be done in one shot :stuck_out_tongue_winking_eye:  but then it's required to keep the `'` along the full command which makes harder for reading and for handling `sos_report_options` when the command is passed to the shell.